release_notes/topics/18_0_0.adoc

```diff
@@ -11,3 +11,10 @@ Recovery Codes as another way to do two-factor authentication is now available a
 For more details, take a look at link:{adminguide_link}#_step-up-flow[{adminguide_name}].
 
 Thanks to https://github.com/CorneliaLahnsteiner[Cornelia Lahnsteiner] and https://github.com/romge[Georg Romstorfer] for the contribution.
+
+== WebAuthn improvements
+
+{project_name} now supports WebAuthn id-less authentication. This allows that WebAuthn Security Key will identify the user during authentication as long as the
```

Comment:
Try to always follow "this" with a noun, so it's clear what "this" is referring to.  This feature or this change..

Reply:
Thanks for the suggestion! It is my favourite mistake :-( I will try to pay more attention next time.